tests/test_mm_pretokenize_masking.py

```
@@ -0,0 +1,121 @@
+from axolotl.utils.chat_templates.base import get_chat_template
+
+MODEL = "Qwen/Qwen2-VL-7B-Instruct"
+
+
+@pytest.fixture(scope="module")
+def qwen2vl_processor():
+    from transformers import AutoProcessor
+
+    try:
+        return AutoProcessor.from_pretrained(MODEL)
+    except Exception as exc:  # pragma: no cover - environment guard
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)